test/e2e/mock-e2e.js

```diff
@@ -865,6 +868,21 @@ async function setupMocking(
       };
     });
 
+  // Nft API: tokens
+  await server
+    .forGet(
+      `https://nft.api.cx.metamask.io/users/${DEFAULT_FIXTURE_ACCOUNT_LOWERCASE}/tokens`,
```

Comment:
:warning: it needs to be lowercase otherwise the url doesn't match and the mock doesn't apply